src/python/dxpy/bindings/apollo/input_arguments_validation_schemas.py

```diff
@@ -106,7 +108,7 @@
         },
         "condition": "with_at_least_one_required",
         "error_message": {
-            "message": '“--expression-matrix" cannot be passed with any argument other than "--retrieve-expression”'
+            "message": '“--expression-matrix" cannot be passed with any argument other than "--retrieve-expression”.'
```

Comment:
Let's use normal quotation marks as `“` might not get correctly outputted by all terminals.

```suggestion
            "message": '"--expression-matrix" cannot be passed with any argument other than "--retrieve-expression".'
```